ponyo/simulate_expression_data.py

```diff
@@ -25,55 +25,49 @@ def fxn():
     fxn()
 
 
-def get_sample_ids(experiment_id, dataset_name, sample_id_colname):
+def get_sample_ids(
+    metadata_filename, delimiter, experiment_colname, experiment_id, sample_id_colname
+):
     """
     Returns sample ids (found in gene expression df) associated with
     a given list of experiment ids (found in the metadata)
 
     Arguments
     ----------
-    experiment_ids_file: str
-        File containing all cleaned experiment ids
+    metadata_filename: str
```

Comment:
It might be useful to link to an example of a metadata file here (e.g. [the one in human_tests](https://github.com/greenelab/ponyo/blob/master/human_tests/data/metadata/recount2_metadata.tsv)).

Reply:
Good tip